kupfer/ui/style.css

```diff
@@ -0,0 +1,30 @@
+#kupfer {
```

Comment:
The current scheme of styling uses only CSS. What could be another alternative to facilitate window styling ? But I should say that CSS styling is easier .